src/Routing/ValidatesRequests.php

```diff
@@ -20,7 +20,7 @@
     public function validate(Request $request, array $rules, array $messages = [], array $customAttributes = [])
     {
         $validator = $this->getValidationFactory()->make($request->all(), $rules, $messages, $customAttributes);
-
+         $validator->setAttributeNames($customAttributes);
```

Comment:
Why do we even have to do this? We are passing custom attributes into the `make` method?